int32 dig_in 
float64 a_in0
float64 a_in1
